.infrastructure/task-definitions/cms_app.json.tpl

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "name": "cds-website-cms",
+    "image": "${image}",
+    "portMappings": [
+      {
+        "containerPort": 1337
+      }
+    ],
+    "environmentFiles": [
+                {    
+                    "value": "arn:aws:s3:::${bucket_name}/.env",
+                    "type": "s3"
```

Comment:
Why the system need to get env from a bucket?

Reply:
If you want to pass a .env file to a taks definition container, AWS forces you to put this file on an S3 bucket. 
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/taskdef-envfiles.html
This statement covers us in case we use the passage of some parameters via .env file and other parameters (sensitive info) via environment var

Reply:
👍 . But if we did not choose this detail I think we should remove this part

Reply:
ok, I'll remove it with the next commit. If we need the env file we will add it again